soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -12,7 +12,7 @@
 #include <soh/Enhancements/custom-message/CustomMessageManager.h>
 #include "soh/Enhancements/item-tables/ItemTableTypes.h"
 
-#define MAX_SEED_BUFFER_SIZE 11 // digits for uint32
+#define MAX_SEED_STRING_SIZE 1023
```

Comment:
Also, it NEEDS to be 1024 here. The limit of 1023 is what we communicate in the tooltip because the 1024th character is the null terminator. Whatever value is here, the number of characters that can actually be used will be one less because of that.

Reply:
For simplicity, we could just have the tooltip itself consume and display this define and just subtract 1 from it, then we wouldn't have to worry about a mismatch.